microceph/cmd/microceph/init.go

```diff
@@ -24,5 +33,146 @@ func (c *cmdInit) Command() *cobra.Command {
 }
 
 func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
-	return fmt.Errorf("Not implemented")
+	// Connect to the daemon.
+	m, err := microcluster.App(context.Background(), c.common.FlagStateDir, c.common.FlagLogVerbose, c.common.FlagLogDebug)
+	if err != nil {
+		return err
+	}
+
+	lc, err := m.LocalClient()
+	if err != nil {
+		return err
+	}
+
+	// Check if already initialized.
+	_, err = lc.GetClusterMembers(context.Background())
+	isUninitialized := err != nil && err.Error() == "Daemon not yet initialized"
```

Comment:
Can we add a sentinel error value for uninitialized cluster?

Reply:
This is a bit of a workaround for now. The goal should be to have a proper boolean API value we can query instead.